cmd/edv-rest/startcmd/start.go

```diff
@@ -349,3 +392,12 @@ func constructCORSHandler(handler http.Handler) http.Handler {
 		},
 	).Handler(handler)
 }
+
+func retry(fn func() error, numRetries uint64) error {
+	return backoff.RetryNotify(fn,
+		backoff.WithMaxRetries(backoff.NewConstantBackOff(sleep), numRetries),
+		func(retryErr error, t time.Duration) {
+			logger.Warnf("failed to connect to database, will sleep for %s before trying again: %s\n",
```

Comment:
The logger will add a newline automatically - you should remove the `\n`

Reply:
(unless you really wanted two line breaks...?)

Reply:
fixed